apps/taglaunch/app.js

```diff
@@ -84,6 +84,15 @@ const unload = () => {
     Bangle.removeListener("lock", lockHandler);
 };
 
+let shortcuts=settings.shortcuts;
+let noShortcuts=shortcuts[0]==""&&shortcuts[1]==""&&shortcuts[2]=="";
+  
+let loadShortcut=function(idx){
+  if(shortcuts[idx]!=""){
+    if(settings.buzz)Bangle.buzz(25);
+    setTimeout(function(){load(shortcuts[idx]+".app.js")},27); //let buzz have effect
```

Comment:
To work exactly when the buzz finishes:

```suggestion
    const p = settings.buzz ? Bangle.buzz(25) : Promise.resolve();
    p.then(() => load(shortcuts[idx]+".app.js"));
```

Reply:
In case there are some hangs, I feel like having a 2ms delay will keep it smooth..
Or I could just be overthinking it.